src/pykeen/ablation/ablation.py

```diff
@@ -128,8 +127,8 @@ def ablation_pipeline(
         if not best_replicates:
             continue
 
-        best_pipeline_dir = os.path.join(output_directory, 'best_pipeline')
-        os.makedirs(best_pipeline_dir, exist_ok=True)
+        best_pipeline_dir = output_directory / "best_pipeline"
```

Comment:
I really don't like the overloading of the `__div__` operator... this makes it so hard to know what this code is supposed to be doing

Reply:
It is meant to be consistent with the way you build paths in unix: `"~/.pykeen"` -> `pathlib.Path("~") / ".pykeen"`

Reply:
There is an alternative: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.joinpath